airflow/providers/papermill/hooks/kernel.py

```diff
@@ -0,0 +1,48 @@
+from airflow.hooks.base import BaseHook
+
+JUPYTER_KERNEL_SHELL_PORT = 60316
+JUPYTER_KERNEL_IOPUB_PORT = 60317
+JUPYTER_KERNEL_STDIN_PORT = 60318
+JUPYTER_KERNEL_CONTROL_PORT = 60319
+JUPYTER_KERNEL_HB_PORT = 60320
+
+
+def register_remote_engine():
+    """
+    Registers ``RemoteKernelEngine`` papermill engine
+    """
+    from papermill.engines import papermill_engines
+    from airflow.providers.papermill.operators.papermill import RemoteKernelEngine, REMOTE_KERNEL_ENGINE
```

Comment:
Hooks doesn't should'n anything from operators. It should be the other way around, Operator imports from Hook(s)

Hooks intend to use:
- Communicate with AWS connections
- Provide generic way to implement integration with service/tool
- Reusable parts for different operators

Reply:
Updated.